### PR TITLE
Add libgcrypt

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

**Comment by martindurant** [comment]
Repeating commit text as a comment: I removed `check` from the build script, because it was taking too long and stalling on CircleCI. Using the same docker image and settings locally, it passes.

This is a follow-up to https://github.com/conda-forge/staged-recipes/pull/3575/ (now at https://github.com/conda-forge/libgpg-error-feedstock ) and required for libgsasl and libhdfs3 to run on systems with incompatible libgcrypt, e.g., ubuntu 16.04.

**Comment by martindurant** [comment]
@ocefpaf , very much the same as https://github.com/conda-forge/staged-recipes/pull/3575

File: recipes/libgcrypt/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+./configure --prefix=$PREFIX
+
+make -j4
```
Comment (ocefpaf) [inline recipes/libgcrypt/build.sh]: Don't hardcode to `-j4`. Use the variable `$CPU_COUNT` instead. (I don't think we have 4 cores in any of the CIs BTW :smile: )

**Comment by ocefpaf** [comment]
One minor thing and this should be good to go. (Use `[skip appveyor]` in your next commit message to avoid that CI. It is quite overloaded at the moment and you don't need it here.)

**Comment by martindurant** [comment]
Sorry, missed the second comment. Is there any way to un-queue appveyor?

**Comment by ocefpaf** [comment]
I can cancel that. Don't worry.

File: recipes/libgcrypt/build.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+./configure --prefix=$PREFIX
+
+make -j$CPU_COUNT
+make install -j$CPU_COUNT
```
Comment (ocefpaf) [inline recipes/libgcrypt/build.sh]: This works @martindurant. No need to remove `-j`.

**Comment by martindurant** [comment]
Thank you!

**Comment by ocefpaf** [comment]
I cancelled Travis-CI too b/c you are only building on Linux.

As soon as CircleCI passes we can merge this.

**Comment by martindurant** [comment]
@ocefpaf , ok done. appveyor still seems to have queued one.

**Comment by ocefpaf** [comment]
That is a bug, if you click on it you can see that it was cancelled.

**Pull request merged**